scripts/screeps_rl_live_dashboard.py

```
@@ -0,0 +1,811 @@
+    tencent_safety = latest_tencent.get("safety") if isinstance(latest_tencent.get("safety"), dict) else {}
+    unsafe_flags: list[JsonObject] = []
+    if latest_tencent:
+        for field in REQUIRED_TENCENT_SAFETY_FIELDS:
+            if field not in tencent_safety:
+                unsafe_flags.append({"source": "tencent", "field": field, "value": "missing"})
+                continue
+            value = tencent_safety[field]
+            if value is not False:
+                unsafe_flags.append({"source": "tencent", "field": field, "value": value})
+    for regression in scorecard.get("safetyRegressions", []):
+        unsafe_flags.append({"source": "scorecard", "field": "safetyRegression", "value": regression})
```

Ruff (0.15.13)

[warning] Use `list.extend` to create a transformed list

(PERF401)